node/src/chain_spec.rs

```diff
@@ -640,11 +704,86 @@ fn foucoco_genesis(
 			..Default::default()
 		},
 		democracy: Default::default(),
-		sudo: foucoco_runtime::SudoConfig { key: Some(sudo_account) },
+		sudo: foucoco_runtime::SudoConfig { key: Some(sudo_account.clone()) },
 		technical_committee: foucoco_runtime::TechnicalCommitteeConfig {
 			members: signatories.clone(),
 			..Default::default()
 		},
+		tokens: foucoco_runtime::TokensConfig {
+			// Configure the initial token supply for the native currency and USDC asset
+			balances: token_balances,
+		},
+		issue: foucoco_runtime::IssueConfig {
+			issue_period: foucoco_runtime::DAYS,
+			issue_minimum_transfer_amount: 1000,
+			limit_volume_amount: None,
+			limit_volume_currency_id: XCM(DOT),
+			current_volume_amount: 0u32.into(),
+			interval_length: (60u32 * 60 * 24).into(),
+			last_interval_index: 0u32.into(),
+		},
+		redeem: foucoco_runtime::RedeemConfig {
+			redeem_period: foucoco_runtime::DAYS,
+			redeem_minimum_transfer_amount: 100,
+			limit_volume_amount: None,
+			limit_volume_currency_id: XCM(DOT),
+			current_volume_amount: 0u32.into(),
+			interval_length: (60u32 * 60 * 24).into(),
+			last_interval_index: 0u32.into(),
+		},
+		replace: foucoco_runtime::ReplaceConfig {
+			replace_period: foucoco_runtime::DAYS,
+			replace_minimum_transfer_amount: 1000,
+		},
+		security: foucoco_runtime::SecurityConfig {
+			initial_status: if start_shutdown {
+				foucoco_runtime::StatusCode::Shutdown
+			} else {
+				foucoco_runtime::StatusCode::Error
+			},
+		},
+		stellar_relay: foucoco_runtime::StellarRelayConfig {
+			old_validators: vec![],
+			old_organizations: vec![],
+			validators,
+			organizations,
+			enactment_block_height: 0,
+			is_public_network: false,
+			phantom: Default::default(),
+		},
+		oracle: foucoco_runtime::OracleConfig {
+			authorized_oracles,
+			max_delay: 3600000, // one hour
+		},
+		vault_registry: foucoco_runtime::VaultRegistryConfig {
+			minimum_collateral_vault: vec![(XCM(DOT), 0), (XCM(KSM), 0)],
+			punishment_delay: foucoco_runtime::DAYS,
+			secure_collateral_threshold: vec![
+				(default_pair(XCM(DOT)), FixedU128::checked_from_rational(260, 100).unwrap()),
+				(default_pair(XCM(KSM)), FixedU128::checked_from_rational(260, 100).unwrap()),
+			], /* 150% */
```

Comment:
The comments with the values are on the wrong lines, and the percentages are wrong. Should be
260%, 200%, and 150% according to the values used.

Reply:
updated/changed